test/Scripts.Integration.Test/Scripts/SmokeTester.cs

```diff
@@ -139,7 +139,7 @@ public static void SmokeTest()
             t.ExpectMessage(currentMessage, "'message':'scope-crumb'}");
             t.ExpectMessage(currentMessage, "'extra':{'extra-key':42}");
             t.ExpectMessage(currentMessage, "'tag-key':'tag-value'");
-            t.ExpectMessage(currentMessage, "'user':{'email':'email@example.com','id':'user-id','ip_address':'::1','username':'username','other':{'role':'admin'}}");
+            t.ExpectMessage(currentMessage, "'user':{'id':'user-id','username':'username','email':'email@example.com','ip_address':'::1','other':{'role':'admin'}}");
```

Comment:
The change of order came from here: https://github.com/getsentry/sentry-dotnet/pull/1920/files

Reply:
Yeah, that should be fine.  Thanks for pointing that out!